packages/awsui-react-ts-website/sample/src/Auth.tsx

```diff
@@ -0,0 +1,121 @@
+/*********************************************************************************************************************
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License").
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ******************************************************************************************************************** */
+
+import React, { createContext, useCallback, useEffect, useState, useMemo } from "react";
+import { Amplify } from "aws-amplify";
+import { Authenticator, ThemeProvider, Theme, useTheme } from "@aws-amplify/ui-react";
+
+/**
+ * Context for storing the runtimeContext.
+ */
+export const RuntimeConfigContext = createContext<any>({});
```

Comment:
Would be nice to define an interface for runtime config :)

Reply:
it does have one, any :)